README.rst

```diff
@@ -64,42 +74,43 @@ Additional optional arguments
+ n_concurrent=<n_concurrent>      \+         \+     Specifies the number of concurrent requests to submit. By default, the script submits four concurrent requests. Set ``<n_concurrent>`` to match the number of cores in the prediction API endpoint.
+ create_api_token                 \+         \+     Requests a new API token. To use this option, you must specify the ``password`` argument for this request (not the ``api_token`` argument). Specifying this argument invalidates your existing API token and creates and stores a new token for future prediction requests.
+ n_retry=<n_retry>                \+         \+     Specifies the number of times DataRobot will retry if a request fails. A value of -1, the default, specifies an infinite number of retries.
+ pred_name=<pred_name>            \+         \+     Applies a name to the prediction column of the output file. If you do not supply the argument, the column name is blank. For binary predictions, only positive class columns are included in the output. The last class (in lexical order) is used as the name of the prediction column.
+ skip_row_id                      \+         \+     Skip the row_id column in output.
+ output_delimiter=<delimiter>     \+         \+     Specifies the delimiter for the output CSV file. The special keyword "tab" can be used to indicate a tab-delimited CSV.
+ timeout=<timeout>                \+         \+     The time, in seconds, that DataRobot tries to make a connection to satisfy a prediction request. When the timeout expires, the client (the batch_scoring or batch_scoring_sse command) closes the connection and retries, up to the number of times defined by the value of ``<n_retry>``. The default value is 30 seconds.
+ delimiter=<delimiter>            \+         \+     Specifies the delimiter to recognize in the input .csv file (e.g., "--delimiter=,"). If not specified, the script tries to automatically determine the delimiter. The special keyword "tab" can be used to indicate a tab-delimited csv.
+ resume                           \+         \+     Starts the prediction from the point at which it was halted. If the prediction stopped, for example due to error or network connection issue, you can run the same command with all the same arguments plus this ``resume`` argument. If you do not include this argument, and the script detects a previous script was interrupted mid-execution, DataRobot prompts whether to resume. When resuming a script, you cannot change the ``dataset_filepath``,  ``model_id``, ``project_id``, ``n_samples``, or ``keep_cols``.
+ help                             \+         \+     Shows usage help for the command.
+ fast                             \+         \+     *Experimental*: Enables a faster .csv processor. Note that this method does not support multiline CSV files.
+ stdout                           \+         \+     Sends all log messages to stdout. If not specified, log messages to the ``datarobot_batch_scoring_main.log`` file.
```

Comment:
If not specified, the command sends log messages to the ``datarobot_batch_scoring_main.log`` file.